docs/introduction/infrastructure.md

```diff
@@ -1,161 +1,104 @@
 # Computational Infrastructure
 
-## Introduction
-One way of understanding the Inductiva API is to see it as an abstraction layer that
-allows you to use computational resources made available by different providers in a
-seamless and uniform fashion. 
+One way to understand the Inductiva API is to see it as an intermediary between 
+users and the complex landscape of computational resources. In this reference, you 
+will learn how the Inductiva API manages your computational workload, and then you'll
+explore the various computational options currently available to you, and additional resources set to be introduced in future updates.
 
-On the server side, Inductiva takes a computational load issued by the user, i.e.,
-one or several simulations, and routes it to a computational resource that the users
-has access to, and takes care of all the orchestration required for running the load
-and returning the results back to the user. 
+## Computational Resource Management
 
-The computational resources effectively used to process that load can potentially be
```

Comment:
I think this part is still efficiently telling the mechanics we provided to verify the best prices, and then we can merge it with the one below the next.